Documentation/ApiOverview/Events/EventDispatcher/Index.rst

```diff
@@ -220,24 +231,25 @@ The PHP attribute :php:`\TYPO3\CMS\Core\Attribute\AsEventListener` supports the
 following properties (which are all optional):
 
 :php:`identifier`
-    A unique identifier must be declared which identifies the event listener,
-    and orderings can be build upon the identifier. It this property is not
-    explicitly defined, the service name is used instead.
+    A unique identifier should be declared which identifies the event listener,
+    and orderings can be build upon the identifier. If this property is not
+    explicitly defined, the service name is automatically used instead.
 
 :php:`before`
     This property allows a custom sorting of registered listeners. The listener
     is then dispatched before the given listener. The value is the identifier of
-    another event listeners.
+    another event listener.
```

Comment:
It can be multiple ones when I remember correctly, separated by comma. So, plural is correct.